src/components/tasks/card/index.tsx

```diff
@@ -354,6 +425,88 @@ const Card: FC<CardProps> = ({
         };
     };
 
+    const onCancelEditRoute = () => {
+        setIsEditMode(false);
+    };
+
+    const CancelEditButton = () => (
```

Comment:
in the tabIndex prop of the Image component. The tabIndex prop is used to control whether the element is focusable or not. A focusable element can be selected with the keyboard and can receive keyboard events. The tabIndex prop should be set to 0 for elements that should be focusable.

In the code you provided, the tabIndex prop of the Image component is set to 0. This means that the Image component is focusable and can receive keyboard events. However, the Image component does not have any functionality that can be controlled with the keyboard. This means that the tabIndex prop of the Image component should be set to -1 to make it non-focusable.

Reply:
Thanks for pointing it out. I have updated tabIndex prop for the Image to -1. 
As this image can be clicked that’s why I added tabIndex of 0 previously.